### PR TITLE
Automatic cleanup!

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "homepage": "https://github.com/PolymerElements/paper-button",
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0-rc.1",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#2.0-preview",
-    "paper-behaviors": "PolymerElements/paper-behaviors#2.0-preview",
-    "paper-styles": "PolymerElements/paper-styles#2.0-preview"
+    "polymer": "Polymer/polymer#1.9 - 2",
```
Comment (valdrinkoshi) [inline bower.json]: awesome 💯

File: bower.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
   "variants": {
     "1.x": {
```
Comment (valdrinkoshi) [inline bower.json]: WDYT of having a 2.x variant where we basically have all the dependency pointing 2.x?

Comment (rictic) [inline bower.json]: I think in practice that's basically the same as the mainline deps, yeah?

Comment (valdrinkoshi) [inline bower.json]: Yep exactly

Comment (valdrinkoshi) [inline bower.json]: Wait, now that I think of it twice: the range will default to the lower version, so basically we're now testing the 1.x dependencies on the mainline deps, no?

Comment (rictic) [inline bower.json]: the range defaults to the highest version afaik

Comment (rictic) [inline bower.json]: closing this to recreate it more better, will link to the recreated version shortly.

Comment (rictic) [inline bower.json]: new version: https://github.com/PolymerElements/paper-button/pull/158

**Pull request closed without merging**